generators/client/templates/react/src/main/webapp/app/shared/reducers/authentication.ts.ejs

```diff
@@ -95,6 +95,32 @@ export const getSession = () => dispatch => dispatch({
   payload: axios.get('/api/account')
 });
 
+<%_ if (authenticationType === 'session') { _%>
+export const login = (username, password, rememberMe = false) => async (dispatch, getState) => {
+  const data = 'j_username=' +
+    encodeURIComponent(username) +
+    '&j_password=' +
+    encodeURIComponent(password) +
+    '&remember-me=' +
+    rememberMe +
+    '&submit=Login';
+  const result = await dispatch({
+    type: ACTION_TYPES.LOGIN,
+    payload: axios.post('/api/authentication', data)
+  });
+
+  const bearerToken = result.value.headers.authorization;
+  if (bearerToken && bearerToken.slice(0, 7) === 'Bearer ') {
```

Comment:
this doesn't look right. Session auth shouldn't have any JWT token. Check the angular `auth-session.service.ts`